models/build.js

```diff
@@ -22,6 +22,7 @@ const STATUSES = [
     'COLLAPSED', // when the build is collapsed
     'FROZEN' // when the build is frozen due to freeze window
 ];
+const STATUS_MESSAGE_TYPES = ['ERROR', 'WARN', 'INFO'];
```

Comment:
```suggestion
const STATUS_MESSAGE_TYPES = ['ERR', 'WARN', 'INFO'];
```
since the other two are shortened, wondering if this is more consistent

Reply:
We don't need to shorten `ERROR`. The values used are pretty much the industry standard.
Ex:
https://github.com/winstonjs/winston?tab=readme-ov-file#using-logging-levels
https://www.tutorialspoint.com/log4j/log4j_logging_levels.htm